CHANGELOG.md

```diff
@@ -5,12 +5,114 @@ Documentation of all notable changes to the **Fizzy** project.
 The format is based on [Keep a Changelog],
 and this project adheres to [Semantic Versioning].
 
+## [0.5.0] — unreleased
+
+This release focuses on behind the scenes code and test cleanups, and preparation for a public API.
+
+Fizzy passes all(*) of the [official test suite (spectest 1.0)](https://github.com/wasmx/wasm-spec/tree/w3c-1.0-jsontests-20200813/test/core/json):
+  - 17911 of 17913 binary parser and execution tests,
+  - 989 of 989 validation tests,
+  - 477 skipped due to testing text format parser.
+
+(*) The two failures are caused by, in our opinion, two bugs in the test cases.
+
+A notable change is the upgrade from wabt 1.0.12 to 1.0.19 in our testing infrastructure.
+While this allows us to conduct fuzzing, it must be noted there is a [significant speed regression](https://github.com/WebAssembly/wabt/issues/1550)
+with the newer wabt.
+
+There is a slight speed regression compared to Fizzy 0.4.0, which can be attributed to ... what exactly?
```

Comment:
@chfast what is the reason behind this? Are these number not with the LTO build and the instantiate/execute split is causing it?

Reply:
Just micro set is omitted.

Reply:
"code layout restructuring in #518".